### ISSUE TITLE
Create a new special intermediary firmware

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-mcu#467]
To make onboarding easier let's create a new firmware, which will update bootloader to its newest version and firmware to the newest version, so user does not have to update (and unplug) its device several times.

**Comment by keepkeyjon** [comment]
You could skip some of the unplugging with soft resets: https://github.com/trezor/trezor-mcu/issues/438

**Comment by saleemrashid** [comment]
Yes, I had a terrible experience attempting to upgrade an old Trezor to the latest firmware (since the new firmwares are too big for the old bootloader).
